ansible/ci-build-collector.yml

```
@@ -70,6 +77,13 @@
         push: true
         source: local
 
+    - name: Push latest to quay.io/stackrox-io
+      community.docker.docker_image:
+        name: "{{ collector_image }}-{{ arch }}-slim"
```

[nit] This doesn't make a difference since all tags will be copies of each other, but if the image we build is the one with no suffix, I feel we should use that one as the base for all these pushes for consistency if nothing else. WDYT?
```suggestion
        name: "{{ collector_image }}-{{ arch }}"
```

---

Yeah fair point - I'll fix this